build.gradle

```diff
@@ -36,6 +36,10 @@ dependencies {
   testImplementation 'org.springframework.boot:spring-boot-starter-test'
 }
 
-test {
+tasks.named('test') {
   useJUnitPlatform()
 }
+
+tasks.named('jar') {
```

Comment:
I don't see this in start.spring.io. What's the rrationale?

Reply:
By default it generates plain-jar alongside uber-jar.
It is not a problem, just unnecessary task and file.